ovara-ui/src/app/lib/configuration.ts

```diff
@@ -0,0 +1,22 @@
+export const DOMAIN =
+  process.env.APP_URL ?? process.env.VIRKAILIJA_URL ?? 'https://localhost:3405';
+
+export const isLocalhost = DOMAIN.includes('localhost');
+
+export const isDev = process.env.NODE_ENV === 'development';
+
+export const isProd = process.env.NODE_ENV === 'production';
+
+export const isTesting = process.env.TEST === 'true';
+
+export const configuration = {
+  raamitUrl: `${DOMAIN}/virkailija-raamit/apply-raamit.js`,
+  loginUrl: `${DOMAIN}/cas/login`,
+  ovaraBackendLogin: `${DOMAIN}/ovara-backend/auth`,
+  sessionCookie: process.env.SESSION_COOKIE || 'JSESSIONID',
+  kayttoikeusUrl: `${DOMAIN}/kayttooikeus-service/henkilo/current/omattiedot`,
```

Comment:
Oletko kutsumassa näitä autentikoitumista vaativia palveluita suoraan nextjs-frontista vaiko kuitenkin ennemmin sieltä scala-backendista? Käyttöoikeudethan tulee siellä spring boot -bäkkärissä jo UserDetailsista loginin yhteydessä joten niitä ei erikseen tarvi hakea apista.